gradle.properties

```
@@ -9,11 +9,11 @@ releasePluginVersion=2.6.0
 puppycrawlCheckstyleVersion=8.18
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.4.0-20230213-151200-6d175fd1
+ballerinaLangVersion=2201.5.0-20230401-003700-e9cbdd41
```

```suggestion
ballerinaLangVersion=2201.4.0-20230213-151200-6d175fd1
```
We don't need to update the lang timestamped version